tool/gravity/cli/update.go

```diff
@@ -51,7 +52,14 @@ func updateTrigger(
 	localEnv *localenv.LocalEnvironment,
 	upgradeEnv *localenv.LocalEnvironment,
 	appPackage string,
-	manual bool) (err error) {
+	manual bool,
+	systemLogFile string,
+) error {
+	err := install.InitLogging(systemLogFile)
```

Comment:
This could be done outside - prior to calling the function. Same for `rpcAgentRun`